cdk ls not working

Hey, 

first: thanks for this wonderful demo! 

When running `bazel run cdk -- ls` I get the following error: 

```
bazel-cdk-demo [master] % bazel run cdk -- ls

INFO: Analyzed target //:cdk (608 packages loaded, 16832 targets configured).
INFO: Found 1 target...
Target //:cdk up-to-date:
  dist/bin/cdk.sh
  dist/bin/cdk_loader.js
  dist/bin/cdk_require_patch.js
INFO: Elapsed time: 27.175s, Critical Path: 5.73s
INFO: 1 process: 1 local.
INFO: Build completed successfully, 19 total actions
INFO: Build completed successfully, 19 total actions
Error: Cannot find module 'com_kindlyops_bazel_cdk_demo/lib/app.js'
Require stack:
-
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:794:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Object.<anonymous> (/private/var/tmp/_bazel_tklinger/ddc144967cbb78a5f9c819c1caa0dd87/execroot/com_kindlyops_bazel_cdk_demo/bazel-out/darwin-fastbuild/bin/lib/app_loader.js:32:24)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)
    at internal/main/run_main_module.js:17:11
Subprocess exited with error 1
```

This happens on a fresh checkout of the master branch. Do I need to change something or is `cdk ls` not yet working?

--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -2,6 +2,10 @@
 # See https://docs.bazel.build/versions/master/build-ref.html#BUILD_files
 
 load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+load("@bazel_gazelle//:def.bzl", "gazelle")
+load("@com_kindlyops_rules_manifest//:manifest.bzl", "docker_manifest", "lambda_manifest")
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+load("@io_bazel_rules_docker//container:container.bzl", "container_push")
 
 # Allow any ts_library rules in this workspace to reference the config
 # Note: if you move the tsconfig.json file to a subdirectory, you can add an alias() here instead
@@ -10,7 +14,6 @@ load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
 exports_files(
     [
         "tsconfig.json",
-        "cdk.json",
         "jest.config.js",
     ],
     visibility = ["//visibility:public"],
@@ -18,14 +21,14 @@ exports_files(
 
 nodejs_binary(
     name = "cdk",
+    args = ["--app $(location //lib:app)"],
     data = [
-        "cdk.json",
         "tsconfig.json",
         "//lib",
         "//lib:app",
-        "@npm//:node_modules",
     ],
     entry_point = "@npm//:node_modules/aws-cdk/bin/cdk",
+    node_modules = "@npm//:node_modules",
     visibility = ["//visibility:public"],
 )
 
@@ -37,12 +40,8 @@ genrule(
     visibility = ["//visibility:public"],
 )
 
-load("@bazel_gazelle//:def.bzl", "gazelle")
-
 # gazelle:prefix github.com/kindlyops/bazel-cdk-demo
 gazelle(name = "gazelle")
-
-load("@com_kindlyops_rules_manifest//:manifest.bzl", "docker_manifest", "lambda_manifest")
 
 lambda_manifest(
     name = "lambdamanifest",
@@ -51,9 +50,6 @@ lambda_manifest(
         "//lambdas/demo2:lambda_zip",
     ],
 )
-
-load("@io_bazel_rules_docker//go:image.bzl", "go_image")
-load("@io_bazel_rules_docker//container:container.bzl", "container_push")
 
 go_image(
     name = "container",

--- a/cdk.json
+++ b/cdk.json
@@ -1,3 +1,0 @@
-{
-  "app": "node lib/app_loader.js"
-}

